src/onboarding/registration/MockedStatSigFeatureFlag.ts

```diff
@@ -0,0 +1,6 @@
+export const shallShowAlternatePlaceholder = () => {
+  return false
+}
+export const shallSkipUsername = () => {
```

Comment:
```suggestion
export const shouldShowSkipUsername = () => {
```
nit: we don't really use "shall" in naming elsewhere, and grammatically it just makes less sense. Also, the username won't necessarily be skipped if this is true, it's just that the button should be shown enabling the user to skip, if they want.

Reply:
remember to update usages if you accept this suggestion